lib/rules/web/admin_console/4.2/status.xyaml

```diff
@@ -27,3 +27,13 @@ check_dim_resource_list:
       xpath: //div[@class='co-disabled']//button[contains(.,'Create')]
   - selector:
       xpath: //div[@class='co-disabled']//div[contains(.,'No <resource_singular>s Found')]
+check_user_starter_guide_message:
+  elements:
+  - selector:
+      visible_text: !ruby/regexp /Welcome to OpenShift/i
+  - selector:
+      visible_text: !ruby/regexp /Create a project for your application/i
+check_get_started_message_when_no_resources:
+  elements:
+  - selector:
+      visible_text: !ruby/regexp /No Workloads Found/i
```

Comment:
Curiously, why we didn't use the existing `check_page_contains` rule?

Reply:
For current 4.2 implementation, only `No Workloads Found` message and can be replaced with 
```
check_get_started_message_when_no_resources:
  params:
    content: No Workloads Found
  action: check_page_match
```

Reply:
Is this what you mean?